cuxhaven

```diff
@@ -1,10 +1,10 @@
 tech-c:
-  - gateway@ffcux.net
+  - moin@freifunk-cuxhaven.de
   
 asn: 65115
 networks:
   ipv4:
-    - 10.115.0.0/19
+    - 10.115.0.0/16
```

Comment:
Ihr könnt gerne ein kürzeres Prefix haben, ich würde jedoch nicht empfehlen ein /16 in einer einzelnen L2 Domäne (https://github.com/Freifunk-Cuxhaven/site-ffcux/blob/ffcux-master/site.conf#L14) zu nutzen, das ist schlicht zu groß was den Noise angeht. Ich denke bei einer einzelnen L2 Domäne immer etwa an ein /20, also 4096 IPv4 Adressen, denn 4000 Endnutzer sind eine größe, die die wenigsten Freifunk-Netze mal eben erreichen.

Reply:
> Ich denke bei einer einzelnen L2 Domäne immer etwa an ein /20, also 4096 IPv4 Adressen, denn 4000 Endnutzer sind eine größe, die die wenigsten Freifunk-Netze mal eben erreichen.

Second that; bei uns sind's derzeit im Peak um 2000 Clients, das macht nur noch bedingt Spaß, sodaß wir derzeit gucken, unsere ~400 Knoten in kleinere Meshes aufzuteilen. Pro Mesh ein /20  erscheint ein guter Wert zu sein, erlaubt auch halbwegs zweckmäßige DHCP-Timeout-Werte.

Reply:
Ping